lib/parser/lexer.rl

```diff
@@ -1367,13 +1527,7 @@ class Parser::Lexer
   expr_variable := |*
       global_var
       => {
-        if    tok =~ /^\$([1-9][0-9]*)$/
-          emit(:tNTH_REF, tok(@ts + 1).to_i)
-        elsif tok =~ /^\$([&`'+])$/
-          emit(:tBACK_REF)
-        else
-          emit(:tGVAR)
-        end
+        global_var
```

Comment:
Could you rename this to `emit_global_var` for consistency please?

Reply:
Addressed in 9e32e7d.